src/__main__.py

```diff
@@ -37,85 +37,121 @@
     "graysexual": [54, 242, 255, 242, 54],
 }
 
+# A dictionary of all the available stats
+stats = {
+    "os": distribution() or system() or 'N/A',
+    "arch": architecture() or 'N/A',
+    "pkgs": packages() or 'N/A',
+    "kernel": kernel() or system() or 'N/A',
+    "uptime": str(timedelta(seconds=clock_gettime(CLOCK_BOOTTIME))).split('.', 1)[0]
+}
+
 # When printed, reset will end the color of the row
 reset = "\033[0m\033[39m"
 
+# When printed, text will become bold
+bold = "\033[1m"
+
 
 def color256(col: int, bg_fg: str) -> str:
     # Alias to avoid manually typing out escape codes every time
     return f"\033[{48 if bg_fg == 'bg' else 38};5;{col}m"
 
 
-def draw_fetch(flag_name: str, width: int = None):
+def generate_fetch(flag_name: str, show_stats=None, width=None):
     # Load the chosen flag from the dictionary of flags
     flag = flags[flag_name]
 
     # Make sure that the row color is different to the color of the hostname
     row_color = color256(flag[1] if flag[0] != flag[1] else flag[2], "fg")
 
-    # The fetch data (system info) to be displayed
-    row_data = [
+    # Set default stats to show in the fetch
+    show_stats = show_stats or ["os", "pkgs", "kernel", "uptime"]
+
+    # Initialise the fetch data (system info) to be displayed with the user@hostname
+    data = [
         f"{color256(flag[0], 'fg') if flag[0] != 0 else color256(242, 'fg')}"
         f"\033[1m{getuser()}@{gethostname()}{reset}",
-        f"{row_color}os      {reset}{distribution() or system() or 'N/A'}",
-        # f"{row_color}arch    {reset}{architecture() or 'N/A'}",
-        f"{row_color}pkgs    {reset}{packages() or 'N/A'}",
-        f"{row_color}kernel  {reset}{kernel() or system() or 'N/A'}",
-        f"{row_color}uptime  {reset}{str(timedelta(seconds=clock_gettime(CLOCK_BOOTTIME))).split('.', 1)[0]}"
     ]
 
+    # Add the chosen stats to the list row_data
+    for stat in show_stats:
+        value = stats[stat]
```

Comment:
```suggestion
        value = stats[stat]()
```
if the stats dictionary uses lambda functions, you'll need to call them